common/types/src/lib.rs

```diff
@@ -68,19 +71,27 @@ pub mod messaging {
 
     // Messages for Balances
 
-    bind_topic!(BalanceEvent<AccountId, Balance>, b"phala/balances/event");
     #[derive(Debug, Clone, Encode, Decode)]
     pub enum BalanceEvent<AccountId, Balance> {
         TransferToTee(AccountId, Balance),
     }
 
-    bind_topic!(BalanceCommand<AccountId, Balance>, b"phala/balances/command");
     #[derive(Debug, Clone, Encode, Decode)]
     pub enum BalanceCommand<AccountId, Balance> {
         Transfer { dest: AccountId, value: Balance },
         TransferToChain { dest: AccountId, value: Balance },
     }
 
+    impl<AccountId, Balance> BalanceCommand<AccountId, Balance> {
```

Comment:
Oh, I think we should call it `Balances` to be aligned with Substrate's Balances pallet

Reply:
If we assume AccountId = sp_runtime::AccountId32, and Balance = u128, do you think it can simplify our code?

Reply:
> If we assume AccountId = sp_runtime::AccountId32, and Balance = u128, do you think it can simplify our code?

If we do so, such types will not be convenient to be used in pallets. Remember the `trait ToH256`?

Reply:
> Oh, I think we should call it `Balances` to be aligned with Substrate's Balances pallet

Done